src/analyze.ts

```diff
@@ -76,9 +76,24 @@ const fsSymbols = {
 };
 const fsExtraSymbols = {
    ...fsSymbols,
+   emptyDir: FS_DIR_FN,
+   emptyDirSync: FS_DIR_FN,
+   ensureDir: FS_DIR_FN,
+   ensureDirSync: FS_DIR_FN,
+   ensureFile: FS_FN,
+   ensureFileSync: FS_FN,
+   ensureSymlink: FS_FN,
+   ensureSymlinkSync: FS_FN,
+   mkdirp: FS_DIR_FN,
+   mkdirpSync: FS_DIR_FN,
+   mkdirs: FS_DIR_FN,
+   mkdirsSync: FS_DIR_FN,
+   pathExists: FS_FN,
+   pathExistsSync: FS_FN,
```

Comment:
I think most of these are not necessary because the represent "writes" but we only want to trace "reads"

Reply:
I think all above my comment should be removed.

Pretty sure `pathExists` is the same as `fs.existsSync()` so you could see if that currently includes assets (I think it doesn't but could be wrong)

Reply:
I [removed most methods](https://github.com/vercel/nft/pull/296/commits/b0b38873c8447e24af212a73752fa877ecfde3ac) but kept `pathExists` / `pathExistsSync` for the symmetry with `fs.exists` / `fs.existsSync`.